src/material/button-toggle/button-toggle.ts

```diff
@@ -574,7 +670,7 @@ export class MatButtonToggle implements OnInit, AfterViewInit, OnDestroy {
   }
 
   /** Whether the toggle is in single selection mode. */
-  private _isSingleSelector(): boolean {
+  _isSingleSelector(): boolean {
```

Comment:
Should we remove "_" since this method is not private?

Reply:
Good catch, will remove it in the next version!

Reply:
Updated the function name.